src/api/spec/features/webui/requests_spec.rb

```diff
@@ -436,4 +436,44 @@
       expect(page).to have_css('.bg-staging')
     end
   end
+
+  describe 'a request with patchinfo' do
+    let(:maintenance_project) do
+      create(:maintenance_project,
+             name: 'MaintenanceProject',
+             title: 'official maintenance space',
+             target_project: [target_project],
+             maintainer: receiver)
+    end
+    let(:maintenance_request) do
+      create(:bs_request_with_maintenance_incident_actions, :with_patchinfo, source_project_name: source_project.name,
+                                                                             source_package_names: [source_package.name],
+                                                                             target_project_name: maintenance_project.name,
+                                                                             target_releaseproject_names: [target_project.name])
+    end
+
+    before do
+      Flipper.enable(:request_show_redesign)
+      login submitter
+      create(:patchinfo, project_name: source_project.name, package_name: 'patchinfo')
+      visit request_show_path(maintenance_request)
+    end
+
+    it 'shows patch information' do
+      expect(page).to have_text('Patches')
+      expect(page).to have_css('#patchinfo-details')
+    end
+
+    it 'shows category badge' do
+      patchinfo_css = find_by_id('patchinfo-details')
```

Comment:
I believe this example could be all combine in one line like so:

`expect(page).to have_css('#patchinfo-details .badge.text-bg-info', text: 'Recommended')`

I didn't give it a try locally.